raster/r.mask.status/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+"""Fixtures for simple sessions"""
+
+import os
+import pytest
+import grass.script as gs
+
+
+@pytest.fixture()
```

Comment:
I would almost say I was adding them because of some warning I got locally, but I can't reproduce that now. Anyway, easy to change everywhere if we decide for a different style later.